bin/clu

```diff
@@ -10,13 +10,27 @@ if ( file_exists( $path = __DIR__ . '/../vendor/autoload.php' )
 	throw new \Exception('Could not locate autoload.php');
 }
 
+\CLU\Checker::check_executables();
+
 $repo_url = ! empty( $argv[1] ) ? $argv[1] : '';
 if ( getenv( 'CLU_GIT_URL' ) ) {
 	$repo_url = getenv( 'CLU_GIT_URL' );
 }
+$repo_local_working_copy = false;
+if ( ! $repo_url ) {
+  $repo_local_working_copy = getcwd();
+  $first_line_of_output = exec('git config --get remote.origin.url', $outputLines, $status);
+  if ( ! $status ) {
+    $repo_url = $first_line_of_output;
+  }
+}
```

Comment:
Given the rest of the project is already using tabs, could we use tabs instead of spaces for indentation?

Reply:
Didn't notice that! I will fix it up.